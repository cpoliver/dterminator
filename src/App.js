import React from 'react';
import { Provider } from 'react-redux';
import { createStore } from 'redux';
import Swiper from 'react-native-swiper';

import reducers from './reducers';
import {
  Decider, DecisionDetail, Home, Profile, Randomizer, Settings
} from './components/pages';

const App = () => (
  <Provider store={createStore(reducers)}>
    <Swiper loop={false} showsPagination={false} index={1}>
      <Randomizer />
      <Swiper loop={false} showsPagination={false} index={1} horizontal={false}>
        <Profile />
        <Home />
        <Settings />
      </Swiper>
      <Decider />
      <DecisionDetail />
    </Swiper>
  </Provider>
);

export default App;
