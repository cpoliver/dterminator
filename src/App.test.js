import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from './App';

const render = () => shallow(<App />);

describe('the App component', () => {
  it('should match the snapshot', () => {
    const component = render();

    expect(toJson(component)).toMatchSnapshot();
  });
});
