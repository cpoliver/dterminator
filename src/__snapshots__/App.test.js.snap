// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the App component should match the snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Connect(Router)
    duration={150}
    scenes={
      Object {
        "decisionDetail": Object {
          "component": [Function],
          "hideNavBar": false,
          "index": 0,
          "key": "decisionDetail",
          "name": "decisionDetail",
          "parent": "root",
          "sceneKey": "decisionDetail",
          "title": "Decision Detail",
          "type": "REACT_NATIVE_ROUTER_FLUX_PUSH",
        },
        "home": Object {
          "component": [Function],
          "hideNavBar": true,
          "index": 0,
          "key": "home",
          "name": "home",
          "parent": "root",
          "sceneKey": "home",
          "type": "REACT_NATIVE_ROUTER_FLUX_PUSH",
        },
        "root": Object {
          "children": Array [
            "home",
            "decisionDetail",
          ],
          "key": "root",
          "leftButtonIconStyle": Object {
            "tintColor": "#25c7ee",
          },
          "name": "root",
          "navigationBarStyle": Object {
            "backgroundColor": "#201b21",
          },
          "parent": undefined,
          "rightButtonIconStyle": Object {
            "tintColor": "#25c7ee",
          },
          "sceneKey": "root",
          "titleStyle": Object {
            "color": "#25c7ee",
            "fontSize": 20,
            "fontWeight": "bold",
          },
          "type": "REACT_NATIVE_ROUTER_FLUX_PUSH",
        },
      }
    }
  />
</Provider>
`;
